docs/plans/2026-02-01-opponent-played-cards-design.md

```
@@ -0,0 +1,873 @@
+# Show Opponent Played Cards During Resolution
+
+> **For Claude:** REQUIRED SUB-SKILL: Use superpowers:executing-plans to implement this plan task-by-task.
+
+**Issue:** https://github.com/pooya-raz/overdrive/issues/8
```

markdownlint-cli2 (0.20.0)

[warning] Bare URL used

(MD034, no-bare-urls)